src/main/java/com/michelin/cio/hudson/plugins/rolestrategy/RoleMap.java

```diff
@@ -506,7 +506,7 @@ protected Boolean hasPermission(Sid p, Permission permission) {
     /**
      * Aborts the iterations.
      * The method can be used from RoleWalker callbacks to preemptively abort the execution loops on some conditions.
-     * @since TODO
+     * @since 2.14
```

Comment:
```suggestion
     * @since 2.10
```

Was introduced in https://github.com/jenkinsci/role-strategy-plugin/commit/f0e117d9b1ea56aec6fed40b4ced110c8b088818